flowdb/sql/04_schema_other.sql

```diff
@@ -68,3 +68,15 @@ Schema used for temp storage during etl.
 */
 
 CREATE SCHEMA IF NOT EXISTS etl;
+
+CREATE TYPE cdrtype AS ENUM ('voice', 'sms', 'mds');
```

Comment:
I'm wondering whether it would be better to use a reference table than an enum, because there is a reasonable chance that we may want to support additional data types in the future? (See e.g. https://tapoueh.org/blog/2018/05/postgresql-data-types-enum/)

We can probably also change this later if required, but we'll likely want to move towards using reference tables in other contexts as well, so this might be a good starting point.

Curious what @greenape thinks.

Reply:
Yeah perhaps but - not now!?

Reply:
If this:

```
ALTER TYPE … ADD VALUE (the form that adds a new value to an enum type) cannot be executed inside a transaction block.
```

is the only disadvantage then I think having an enum is simpler...

Reply:
Hmm, it might be. Not sure if there are any issues with tables being locked for a while if an enum is updated, but in any case we can tackle this (if needed) when we re-think the other parts of the schema.